packages/application/src/shell.ts

```diff
@@ -87,7 +89,8 @@ export namespace ILabShell {
     | 'menu'
     | 'left'
     | 'right'
-    | 'bottom';
+    | 'bottom'
+    | 'down';
```

Comment:
For extension authors, we'll probably want to document this in the extension developer guide too: https://jupyterlab.readthedocs.io/en/stable/extension/extension_points.html#jupyter-front-end-shell

Reply:
Thanks for the suggestion @jtpio. I updated the doc.